### DISCUSSION TITLE
issue: Cannot access and display error message of the field input which was created with useFieldArray because of type error

### DISCUSSION THREAD

**Discussion by aabduvakhobov02** [react-hook-form/react-hook-form#10598] [Q&A] [answered]
### Version Number

7.43.5

### Codesandbox/Expo snack

https://codesandbox.io/s/busy-swartz-4jyww5?file=/src/App.tsx

### Steps to reproduce

This works perfectly fine in CodeSandbox, but whenever I want to build it to my dev environment it throws error of: 
![image](https://github.com/react-hook-form/react-hook-form/assets/56033647/967b861f-367e-4430-8b25-bf38db5499dd)

Technologies I am using: Next.js, typescript, vercel for deployment

### Expected behaviour

Displaying validation error messages whithout errors

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment, marked as answer]
Try to infer type or provide generic at `useForm`, and run type check again your dev build
